Objects/typeobject.c

```diff
@@ -9114,4 +9161,5 @@ PyTypeObject PySuper_Type = {
     PyType_GenericAlloc,                        /* tp_alloc */
     PyType_GenericNew,                          /* tp_new */
     PyObject_GC_Del,                            /* tp_free */
+    .tp_vectorcall = (vectorcallfunc)super_vectorcall,
```

Comment:
Would you like to refactor by using modern C style initialization for all attributes? (tp_new, tp_alloc, ....)

Reply:
I usually leave it as is. Some people dislike it when I convert the other attributes. I'm aware some people want to convert these too.

Reply:
https://github.com/python/cpython/pull/30820#r790278098

At least, I and Victor love to use it :) But it's up to you